notebooks/lectures/Introduction_to_Python/preview.html

```diff
@@ -1,8 +1,10 @@
-<head>
-  <meta charset="utf-8" />
-  <title>Cloned from "Quantopian Lecture Series: Introduction to Python"</title>
+<!DOCTYPE html>
```

Comment:
Any idea how this file was generated?

Reply:
I just upload an .html version (in addition to .ipynb) of the notebook to this folder. same way we've always updated the files in this repo in my experience

Reply:
> Any idea how the preview.html files were generated? Some of the titles look different and I just want to make sure that we expected this or whether the previews should remain unchanged.

What are the differences you're seeing? The .html previews should look identical to the .ipynb files in terms of content

Reply:
It looks like the title might have changed, at least?